Rubberduck.Core/UI/CodeExplorer/Commands/ImportCommand.cs

```diff
@@ -86,9 +89,53 @@ protected override void OnExecute(object parameter)
 
             foreach (var filename in _openFileDialog.FileNames)
```

Comment:
This is not new in this PR, but while you are in here, could you please wrap this loop in a suspension action? As it currently stands, each module import will start a separate parse, potentially invalidating the project reference. (You can find an example in the `TestEngine`.)

Reply:
Suspension action? I didn't see it in [TestEngine.cs](https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.UnitTesting/UnitTesting/TestEngine.cs). This is something I'm not familiar with.

Reply:
Hou can request go run something in a suspended parser state on the `RubberDuckParserState`. @bclothier introduced this a while ago and the test engine is using it to prevent a reparse while running tests.

It is not ghost important to add this right now. It could be added in aother PR.